spec/factories/articles.rb

```diff
@@ -5,6 +5,7 @@
     transient do
       title { generate :title }
       published { true }
+      published_at { Time.now }
```

Comment:
tiny request: we favor `Time.current` over `Time.now` as it brings with itself the timezone information.

Also FYI: we use `timestamp without timezone` as almost all Rails apps do unfortunately but tests run in a random timezone: https://github.com/forem/forem/blob/d2d9984f28b1d0662f2a858b325a0e6b7a27a24c/spec/spec_helper.rb#L3

Reply:
I haven't found any meaningful difference between them as long as you set `ENV['TZ']` and `Rails.application.config.time_zone` to the same value (and it makes me wonder why Rails still has `ActiveSupport::TimeWithZone` at all because [we get the `TZ` env var functionality from libc](https://www.gnu.org/software/libc/manual/html_node/TZ-Variable.html)), so normally I do that instead of standardizing on `Time.current` over `Time.now` so that it doesn't put the responsibility on individuals.

However, I looked at the code for Zonebie and [it only sets _one_ of them](https://github.com/alindeman/zonebie/blob/0a9e08f3b73cae5979c61e55213d779e6489fb91/lib/zonebie.rb#L7-L18). So we _do_ need to pick one for now. 🙃

> Also FYI: we use `timestamp without timezone` as almost all Rails apps do

Of all my pet peeves with Rails, the fact that ActiveRecord uses this column type for timestamps bugs me the most 😄